test/test_pyquat.py

```diff
@@ -257,9 +258,9 @@ def test_expm(self):
         B1    = 13/51.0
         dt    = 0.05
         w     = np.array([[0.03, 0.02, 0.01]]).T
-        J     = np.diag([200.0, 200.0, 100.0])
-        J_inv = linalg.inv(J)
-        wk1   = pq.wdot(w, J)
+        # J     = np.diag([200.0, 200.0, 100.0])
+        # J_inv = linalg.inv(J)
+        # wk1   = pq.wdot(w, J)
```

Comment:
I believe this was supposed to test code that is used by `expm` before testing `expm` itself.

Reply:
Is it still? these variables are unused.